### ISSUE TITLE
Mask that can have only one character, and others number?

### ISSUE THREAD

**Issue by PokerAsh** [egslava/edittext-mask#21]
Hey,
I want to make this type of mask: A01.02.003
Is it possible?

**Comment by egslava** [comment]
No, though it's a really nice feature to have. Please, if you have a wish to help me with that - please, don't hesitate to write, I'll tell you what exactly you can do to help the project! :)

**Comment by egslava** [comment]
Linked issue: #26. Sunisha, please, elaborate.

**Comment by egslava** [comment]
@PokerAsh could you, also, please, provide your case? What were you trying to enter to the text field? I belive, we could use it in FAQ/Documentation.
